Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -101,6 +101,11 @@
 #string STR_SLEEP_TYPE_S0IX   #language en-US "Suspend to Idle (S0ix)"
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 
+
+#string STR_WIFI_BT_ENABLE_PROMPT    #language en-US "Enable Wi-Fi + BT radios"
+#string STR_WIFI_BT_ENABLE_HELP      #language en-US "When not enabled, WiFi + Bluetooth card is completely disconnected from power."

```

Comment:
```suggestion
#string STR_WIFI_BT_ENABLE_HELP      #language en-US "When not enabled, WiFi + Bluetooth card slot is disabled."


The statement is not correct. From the code in coreboot we disable only the IP blocks responsible for communication with WiFi/BT card, not cutting off power from it. The WiFi/BT card power is controlled by EC. If our goal was to cut off the power from WiFI/BT card it would be better to add an EC cmd to disable WiFI/BT card power during boot. That way we would not need to disable the IP blocks (PCIe, CNVI and USB)
```

Reply:
Applied